pkg/storage/driver/cfgmaps.go

```diff
@@ -166,6 +166,8 @@ func (cfgmaps *ConfigMaps) Create(key string, rls *rspb.Release) error {
 	if _, err := cfgmaps.impl.Create(context.Background(), obj, metav1.CreateOptions{}); err != nil {
 		if apierrors.IsAlreadyExists(err) {
 			return ErrReleaseExists
+		} else if apierrors.IsRequestEntityTooLargeError(err) {
+			return ErrReleaseStorageLimit
```

Comment:
I see how you followed the pattern for returning an error just like the one above. The API is returning an error with the limit amount. Wouldn't it be better to pass on the limit k8s is returning?

Reply:
Try to wrap kubernetes' error return information directly?
Example:
```
release: storage limit, Request entity too large: limit is 3145728
```